bindings/nodejs/lib/types/wallet/bridge/account.ts

```diff
@@ -97,8 +97,8 @@ export type __DeregisterParticipationEventMethod__ = {
     };
 };
 
-export type __GenerateAddressesMethod__ = {
-    name: 'generateAddresses';
+export type __GenerateEd25519AddressesMethod__ = {
+    name: 'generateEd25519Addresses';
```

Comment:
And here? Sorry for the spam, just close them if it doesn't make sense, just wondering why it doesn't appear here.